cypress/support/pages/admin_console/manage/clients/ClientRolesTab.ts

```diff
@@ -1,17 +1,35 @@
-export default class ClientRolesTab {
+import CommonPage from "../../../CommonPage";
+
+enum ClientRolesTabItems {
+  details = "Details",
+  attributes = "Attributes",
+  usersInRole = "Users in role",
```

Comment:
Can we use the same convention as TypeScript?
```suggestion
  Details = "Details",
  Attributes = "Attributes",
  UsersInRole = "Users in role",
```